aio/content/examples/rx-library/src/simple-creation.2.ts

```diff
@@ -0,0 +1,22 @@
+// #docplaster
+/*
+  Because of how the code is merged together using the doc regions,
+  we need to indent the imports with the function below.
+*/
+/* tslint:disable:align */
+// #docregion interval
+  import { interval } from 'rxjs';
+
+// #enddocregion interval
+
+export function docRegionInterval(console) {
+  // #docregion interval
+  // Create an Observable that will publish a value on an interval
+  const secondsCounter = interval(1000);
+  // Subscribe to begin publishing values
+  const subscription = secondsCounter.subscribe(n =>
+    console.log(`It's been ${n} seconds since subscribing!`));
```

Comment:
As per https://github.com/angular/angular/pull/38905#discussion_r491486940, this should actually be:

```suggestion
    console.log(`It's been ${n + 1} seconds since subscribing!`));
```